reporting/accuracy/plot_jacobi_theta2.cpp

```diff
@@ -7,15 +7,14 @@
 using boost::math::tools::ulps_plot;
 
 int main() {
-    using PreciseReal = long double; // boost::multiprecision::float128;
-    using CoarseReal = double;
+    using PreciseReal = long double;
+    using CoarseReal = float;
 
-    // TODO Small q is fine but larger q has pretty bad ULPs
     auto jacobi_theta_coarse = [](CoarseReal z) {
-        return boost::math::jacobi_theta2<CoarseReal>(z, 0.05);
+        return boost::math::jacobi_theta2<CoarseReal>(z, 0.9);
     };
     auto jacobi_theta_precise = [](PreciseReal z) {
-        return boost::math::jacobi_theta2<PreciseReal>(z, 0.05);
+        return boost::math::jacobi_theta2<PreciseReal>(z, 0.9);
```

Comment:
0.9 is somewhat of a bad choice, since it's parsed as a double and then promoted to long double or parsed as double and cast back to float. That adds a source of inaccuracy that is uninteresting in this context. I'd use a representable here, like 15/16.

Reply:
Good catch, will fix.

Reply:
Fixed in latest commit.